app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,24 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag url: '/login', layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-4" do |f|
```

Comment:
I would not recommend hardcoding paths like `url: '/login'` - use Rails named routes for that purpose.